themes/src/main/resources-community/theme/base/account/messages/messages_pl.properties

```diff
@@ -1 +1,242 @@
+
+accountManagementWelcomeMessage=Witamy w zarządzaniu kontem
+personalInfoIntroMessage=Zarządzaj informacjami podstawowymi o sobie
+accountSecurityTitle=Bezpieczeństwo Konta
+accountSecurityIntroMessage=Kontroluj swoje hasło i dostęp
+applicationsIntroMessage=Śledź i zarządzaj uprawnieniami aplikacji do twojego konta
+resourceIntroMessage=Udostępnij swoje zasoby członkom zespołu
+passwordLastUpdateMessage=Twoje hasło zostało zaktualizowane
+updatePasswordTitle=Aktualizuj hasło
+updatePasswordMessageTitle=Miej pewność, że wybrałeś silne hasło
+updatePasswordMessage=Silne hasło zawiera mieszaninę cyfr, liter i symboli. Nie używaj zwykłych słów oraz haseł używanych na innych kontach.
+personalSubTitle=Twoje dane osobiste
+personalSubMessage=Zarządzaj informacjami podstawowymi: twoim imieniem, nazwiskiem oraz emailem
+
+authenticatorCode=Kod jednorazowy
+email=Email
+firstName=Imię
+givenName=Imię
+fullName=Pełna nazwa
+lastName=Nazwisko
+familyName=Nazwisko rodowe
+password=Hasło
+currentPassword=Aktualne hasło
+passwordConfirm=Potwierdzenie
+passwordNew=Nowe hasło
+username=Nazwa użytkownika
+address=Adres
+street=Ulica
+locality=Miejscowość
+region=Stan, województwo, region
+postal_code=Kod pocztowy
+country=Kraj
+emailVerified=Email zweryfikowany
+gssDelegationCredential=Poświadczenia delegowane GSS
+
+profileScopeConsentText=Profil użytkownika
+emailScopeConsentText=Adres email
+addressScopeConsentText=Adres
+phoneScopeConsentText=Telefon
+offlineAccessScopeConsentText=Dostęp offline
+samlRoleListScopeConsentText=Moje role
+rolesScopeConsentText=Role użytkownika
+
+role_admin=Admin
+role_realm-admin=Strefa Admin
+role_create-realm=Utwórz strefę
+role_view-realm=Przeglądaj strefy
+role_view-users=Przeglądaj użytkowników
+role_view-applications=Przeglądaj aplikacje
+role_view-clients=Przeglądaj klientów
+role_view-events=Przeglądaj zdarzenia
+role_view-identity-providers=Przeglądaj dostawców tożsamości
+role_view-consent=Przeglądaj zgody
+role_manage-realm=Zarządzaj strefami
+role_manage-users=Zarządzaj użytkownikami
+role_manage-applications=Zarządzaj aplikacjami
+role_manage-identity-providers=Zarządzaj dostawcami tożsamości
+role_manage-clients=Zarządzaj klientami
+role_manage-events=Zarządzaj zdarzeniami
+role_view-profile=Przeglądaj profil
+role_manage-account=Zarządzaj kontem
+role_manage-account-links=Zarządzaj linkami konta
+role_manage-consent=Zarządzaj zgodami
+role_read-token=Odczytaj token
+role_offline-access=Dostęp offline
+role_uma_authorization=Uzyskaj uprawnienia
+client_account=Konto
+client_account-console=Konsola konta
+client_security-admin-console=Konsola administratora bezpieczeństwa
+client_admin-cli=Admin CLI
+client_realm-management=Zarządzanie strefą
+client_broker=Broker
+
+
+requiredFields=Wymagane pola
+allFieldsRequired=Wszystkie pola są wymagane
+
+backToApplication=&laquo; Powrót do aplikacji
+backTo=Wróć do {0}
+
+date=Data
+event=Zdarzenie
+ip=IP
+client=Klient
+clients=Klienci
```

Comment:
@bdaw I always have a problem with this one: "Klienci" or "Klienty" ? Usually, I use "Aplikacje klienckie", so I don't hit this problem.